rules.md

```diff
@@ -17,17 +17,17 @@ View [RULES HISTORY](https://github.com/dashincubator/dash-incubator-app/commits
+[Dash Incubator](https://dashincubator.app/) is an open-source, blockchain-funded organization that redistributes [Dash](https://www.dash.org/) issued from [Superblocks](https://docs.dash.org/en/stable/governance/understanding.html) to users who produce open-source output that supports our [Mission](#11-mission) and [Vision](#12-dash-vision).
 
-The Incubator uses its own micro-incentive based [Bounty](#2-bounties) system to fund and manage the end-to-end delivery of a wide variety of projects [proposed](#21-concepts) by the community, ranging from small [jobs](#22-bounty-types) such as bug bounties, to ongoing [services](#22-bounty-types) such as promotions or infrastructure provision, to mid-size development [projects](#22-bounty-types) such as dApps and development tools, to large development [programmes](#22-bounty-types) involving multiple products that need to be developed and launched together.
+The Incubator uses a [Bounty](#2-bounties) system to fund and manage the end-to-end delivery of a wide variety of projects [proposed](#21-concepts) by the community, ranging from small [jobs](#22-bounty-types) such as bug bounties, to ongoing [services](#22-bounty-types) such as promotions or infrastructure provision, to mid-size development [projects](#22-bounty-types) such as dApps and development tools, to large development [programmes](#22-bounty-types) involving multiple products that need to be developed and launched together.
 
-Fund allocation and management is decentralized across [Admin](#4-admins) users who are [incentivized](#44-admin-rewards) with a percentage of the Dash they award on Bounties and a [PO role](#53-proposal-owner) responsible for fund management.
+Fund allocation and management is decentralized across [Admin](#4-admins) users and [Strategists](#46-strategists).
 
-The Incubator is classified as an [open-source Dash Funded Organization (DFO)](#14-full-transparency), meaning that all internal operations of the Incubator are conducted in the public domain at all times.  This includes how work is proposed, selected, specified, developed, tested, approved, which users are completing or managing work, what rewards they are allocating or receiving and all internal decision making and governance.
+The Incubator is a [Dash Funded Organization (DFO)](#14-full-transparency).  All internal operations of the Incubator are conducted in the public domain at all times.  This includes how work is proposed, selected, specified, developed, tested, approved, which users are completing or managing work, what rewards they are allocating or receiving and all internal decision making and governance.
```

Comment:
> All internal operations of the Incubator are conducted in the public domain at all times.

Is this technically correct with so many project conversations within private group chats?

Reply:
This particular text was from Andy.  I agree it's not precise in terms of how we operate. 

in this PR I was trying to focus on the substantial changes I was proposing and not review and update all of Andy's original text, even if I disagreed with it.

I think the whole document needs to be reviewed for accuracy, and I'd personally like to see it reduced to at least half its size, but that would be a second pass.